src/main/webapp/scripts/map.js

```diff
@@ -12,31 +12,51 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/* Global Icon Variables */
+var yellowIcon;
+var blueIcon;
+var purpleIcon;
+var redIcon;
+var greyIcon;
 
+/* Hard-coded Recommendations */
+const recommendations = [
+  ['Places to Visit', 'Sky Garden', '51.510881, -0.083751', 'Sky Garden is a great place to visit, with really good views of London. \
+  You can enjoy the 360 degree view of the city - for free! They are public gardens and definitely worth a visit \
+  when you have the time. Remember to keep socially distanced!'],
+  ['Restaurants', 'Mildreds Kings Cross', '51.531299, -0.117168', 'A lovely little place to grab something to eat! There are a lot of \
+  different options and a great atmosphere inside Mildred\'s!'],
+  ['Bars and Clubs', 'Fitz\'s Bar', '51.517735, -0.097112', 'This is a recommendation for Fitz\'s Bar in London.'],
+  ['Study Places', 'Bloomsbury Coffee House', '51.525212, -0.126469', 'A really great place to study!']
+]
 
-/* Custom Markers */
-const yellowIcon = {
-  url: "/images/yellow-marker.png",
-  scaledSize: new google.maps.Size(30, 40),
-};
-
-const blueIcon = {
-  url: "/images/blue-marker.png",
-  scaledSize: new google.maps.Size(30, 40),
-};
-
-const purpleIcon = {
-  url: "/images/purple-marker.png", 
-  scaledSize: new google.maps.Size(30, 40),
-};
-
-const redIcon = {
-  url: "/images/red-marker.png", 
-  scaledSize: new google.maps.Size(30, 40),
-};
+window.onload = function() {
+  /* Custom Markers */
+  yellowIcon = {
+    url: "/images/yellow-marker.png",
+    scaledSize: new google.maps.Size(30, 40),
```

Comment:
```suggestion
    scaledSize: new google.maps.Size(30, 40)
```
same with the other icons